spec/std/tuple_spec.cr

```diff
@@ -341,4 +341,15 @@ describe "Tuple" do
     ary = Tuple.new.to_a
     ary.size.should eq(0)
   end
+
+  it "#to_static_array" do
```

Comment:
There should be a test for tuples with downcasted elements (e.g. `Tuple(Int32 | String).new(1)`)